src/app/story/directives/status.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { cit, create, provide, stubPipe } from '../../../testing';
 import { Router } from '@angular/router';
 import { Angulartics2 } from 'angulartics2';
 import { RouterStub } from '../../../testing/stub.router';
-import { ModalService, ToastrService } from '../../core';
+import { ToastrService, ModalService } from '../../core';
```

Comment:
are we now deciding to import everything directly from styleguide? if so, i definitely don't think you need to fix every instance here - but maybe we can throw that in a low-priority ticket for later.